### ISSUE TITLE
Unable to install

### ISSUE THREAD

**Issue by Issacar13** [tianevitt/tianzes-gauntlight-extras#1]
Maybe it's just me who doesn't understand how to install this, But I can't seem to figure it out and it does not appear in the module list within foundry either.

<img width="458" alt="image" src="https://github.com/tianevitt/tianzes-gauntlight-extras/assets/168503811/191d2286-6c79-428b-96b7-e047c867284c">

**Comment by tianevitt** [comment]
Hey there! I should try to mark this as private because it is not quite ready yet. I put it here so I could install it on a separate test machine, which revealed some issues that I have since fixed. 

It won't install from Foundry because Foundry does not know about it, and because it does not yet have a real release package. If you want to download it using the module.json in the parent folder, well I did get that to work, but I warn you, it is very unpolished, and I have since fixed a lot of game rule issues, and nothing is zipped.

I am now trying to get it polished up so I can actually release it, but I have had a number of illnesses over the last few months, so it is slow going. I work on it almost every day.

I appreciate you leaving this comment anyway; I shall use it as inspiration to get this done.
